test/controllers/ProjectExplorerControllerSpec.scala

```diff
@@ -118,4 +118,57 @@ class ProjectExplorerControllerSpec extends PlaySpec with OneAppPerSuite with Mo
       graphProjectOwner mustBe contentProjectOwner
     }
   }
+
+  "The project resources query" should {
+    "return all buckets in a project if resource=bucket" in {
+      val projectId = g.V().has( Constants.TypeKey, "project:project" ).asScala.toList.head.id()
+      val t = g.V( projectId ).inE( "project:is_part_of" ).otherV().has( Constants.TypeKey, "resource:bucket" ).asScala.toList
+
+      val result = projectController.retrieveProjectResources( projectId.toString.toLong, Some( "bucket" ) ).apply( fakerequest )
+      val content = contentAsJson( result ).as[List[PersistedVertex]]
+
+      content.length mustBe t.length
+    }
+  }
+
+  "The project resources query" should {
+    "return all files attached to a project if resource=None" in {
```

Comment:
files -> nodes ? or maybe resources ?

Reply:
Oh that is just a bad description. It should read all nodes. Will fix.